### PR TITLE
1.19.3 protocol updates

### PR DESCRIPTION
**Pull request by frej4189**

### CODE REVIEW

File: data/pc/1.19.3/protocol.json [hunk 1]
```diff
@@ -3707,7 +3707,7 @@
             },
             {
               "name": "flags",
-              "type": "i8"
+              "type": "u8"
```
Comment (extremeheat) [inline data/pc/1.19.3/protocol.json:3710]: Why the data type changes? Does it fix anything broken? If it's applicable to the previous versions you'd have to go back and change all the previous versions' types also.

Comment (frej4189) [reply data/pc/1.19.3/protocol.json:3710]: I went through the entire protocol packet by packet and just changed the types as I came across them. As for the ubyte/byte that probably won't make any difference as they usually only require the LSBs, but I figured it makes more sense to use the same types as the Vanilla client.

Comment (rom1504) [reply data/pc/1.19.3/protocol.json:3710]: Do previous versions need the same change?

Comment (frej4189) [reply data/pc/1.19.3/protocol.json:3710]: To match Vanilla implementation, yes.

Comment (rom1504) [reply data/pc/1.19.3/protocol.json:3710]: then can you please change it all ? (or revert for this version)

**Comment by rom1504** [comment]
Let's not

**Comment by frej4189** [comment]
This PR also fixes other things. Closing it because of a semantic disagreement doesn't really make sense

**Comment by rom1504** [comment]
Ah yes I didn't see the sound event part

Could you revert the i8/u8 part ? (Or apply it in other versions)

**Comment by TheRedXD** [comment]
I fixed the soundEvent issue in 1.19.3, 1.19.4 and 1.20
The data type change is also not there, for now at least
This is the pull request that changes it: https://github.com/PrismarineJS/minecraft-data/pull/731

**Comment by rom1504** [comment]
I merged the other PR

**Pull request closed without merging**